src/Board.hpp

```diff
@@ -1,7 +1,9 @@
 #pragma once
+#include "FieldsEnum.hpp"
 #include "json.hpp"
 #include "FieldsEnum.hpp"
 #include <vector>
+#include <iostream>
```

Comment:
iostream is not needed in hpp. cout is used in cpp file, so it should be moved to Board,cpp

Reply:
moved